src/psyclone/psyir/transformations/omp_loop_trans.py

```diff
@@ -228,7 +228,9 @@ def _directive(self, children, collapse=None):
             node.reprod = self._reprod
         return node
 
-    def apply(self, node, options=None):
+    def apply(self, node, options=None,
+              reprod: bool = Config.get().reproducible_reductions,
```

Comment:
This use of Config() may have unintended consequences as it will instantiate it when this module is imported and that *may* be before any custom config file has been parsed? @hiker is probably best placed to comment on this.

Reply:
Thats fine - I can change this to be None and do an `if reprod is None` check after to initialise it when this file is called.

Reply:
Changed.